### PR TITLE
Release notes for 0.13.1

### PR DESCRIPTION
**Pull request by mkolesnik**
Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.1
+
+This is a bugfix release:
+
+* Allow broker certificate checks to be disabled for insecure connections, using `subctl join --check-broker-certificate=false`.
+* Return local cluster IP for headless services.
+* Display proper output message on `subctl show brokers` when broker is not installed on the cluster.
+* Allow passing `DEFAULT_REPO` while building subctl.
+* Cleanup the host routes programmed by OVN RA plugin during uninstall.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Clean up the host routes programmed by OVN RA plugin during uninstall.
```

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
+* Address overrides by component. This avoid any problems downstream, in case that the image names are
+getting rewritten.
+* Limit Azure machine name lengths to 40 characters.
+* Mention default cable driver in join help msg.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Document the default cable driver in the `subctl join` help message.
```

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
+* Set DNSPolicy to ClusterFirstWithHostNet - for pods that run with `HostNetworking: true`.
+* Remove hardcoded workerNodeList while querying image for GCP and RHOS cloud preparation steps.
+* Collect the output of `ovn-sbctl` show in `subctl gather`.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Collect the output of `ovn-sbctl show` in `subctl gather`.
```

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
+
+This is a bugfix release:
+
+* Allow broker certificate checks to be disabled for insecure connections, using `subctl join --check-broker-certificate=false`.
+* Return local cluster IP for headless services.
+* Display proper output message on `subctl show brokers` when broker is not installed on the cluster.
+* Allow passing `DEFAULT_REPO` while building subctl.
+* Cleanup the host routes programmed by OVN RA plugin during uninstall.
+* Address overrides by component. This avoid any problems downstream, in case that the image names are
+getting rewritten.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Support overriding image names per-component to better support downstream builds.
```

File: src/content/community/releases/_index.en.md [hunk 5]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.1
+
+This is a bugfix release:
+
+* Allow broker certificate checks to be disabled for insecure connections, using `subctl join --check-broker-certificate=false`.
+* Return local cluster IP for headless services.
+* Display proper output message on `subctl show brokers` when broker is not installed on the cluster.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Display proper output message from `subctl show brokers` when broker is not installed on the cluster.
```

File: src/content/community/releases/_index.en.md [hunk 6]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
+* Allow passing `DEFAULT_REPO` while building subctl.
+* Cleanup the host routes programmed by OVN RA plugin during uninstall.
+* Address overrides by component. This avoid any problems downstream, in case that the image names are
+getting rewritten.
+* Limit Azure machine name lengths to 40 characters.
+* Mention default cable driver in join help msg.
+* Set DNSPolicy to ClusterFirstWithHostNet - for pods that run with `HostNetworking: true`.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Set `DNSPolicy` to `ClusterFirstWithHostNet` for pods that run with `HostNetworking: true`.
```

File: src/content/community/releases/_index.en.md [hunk 7]
```diff
@@ -4,6 +4,24 @@ title = "Releases"
...
+* Remove hardcoded workerNodeList while querying image for GCP and RHOS cloud preparation steps.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Remove hardcoded `workerNodeList` while querying image for GCP and RHOS cloud preparation steps.
```

**Review by dfarrell07 — CHANGES_REQUESTED** [review]

**Comment by mkolesnik** [comment]
Applied Daniel's suggestions, let's make sure to **squash when merging**.

**Review by dfarrell07 — APPROVED** [review]

**Comment by dfarrell07** [comment]
Squashed and merged, thanks Mike!

**Pull request merged**